packages/beacon-node/src/network/discv5/index.ts

```diff
@@ -112,6 +116,12 @@ export class Discv5Worker extends (EventEmitter as {new (): StrictEventEmitter<E
     }
   }
 
+  async discoverKadValues(): Promise<void> {
+    if (this.status.status === "started") {
+      await this.status.workerApi.discoverKadValues();
```

Comment:
The `this.status.status` looks really weird. Can we rename it to `this.status` to `this.statusWorker` and `Discv5WorkerStatus` to `Discv5Worker`.